ui/src/kubernetes/components/KubernetesDashboard.js

```diff
@@ -73,7 +82,11 @@ export const KubernetesPage = React.createClass({
               <h1>Kubernetes Dashboard</h1>
             </div>
             <div className="page-header__right">
-              <h1>Range:</h1>
+              <div className="btn btn-info btn-sm" data-for="graph-tips-tooltip" data-tip={this.renderGraphTips()}>
```

Comment:
I really don't understand what's happening here. hmmm, putting a markup string in a data attribute of another html tag? haha so weird